.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publisher
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          
+      - name: Run script
+        env:
+          REPO_TOKEN: ${{ secrets.CLIN_FHIR_DEPLOY_KEY }}
+        run: |
+          SCRIPT=.github/build.sh
+          if [ -f $SCRIPT ]; then 
+            chmod u+x $SCRIPT
+            $SCRIPT
+          fi
```

I would remove the 'if' condition here and just run the script.

If the script is missing, it should not silently do nothing. Rather, the pipeline should be allowed to fail as something is clearly wrong (either the script should be re-added or the pipeline should also be deleted at that point)